phpstan.neon

```diff
@@ -3,10 +3,9 @@ includes:
 	- vendor/phpstan/phpstan-phpunit/extension.neon
 	- vendor/phpstan/phpstan-phpunit/rules.neon
 parameters:
-	# TODO remove these when working on the payment tickets
 	excludePaths:
 		analyse:
-			- src/Domain/Model/PayPalPayment.php
+			- tests/cli
```

Comment:
With the split of command definition and execution, this might no longer be necessary. If PHPStan still executes, then just exclude the execution file.